docs/dockerfile_experimental.md

```diff
@@ -0,0 +1,128 @@
+# Dockerfile frontend experimental syntaxes
+
+The contents of this document is planned to be moved to https://docs.docker.com in future.
+
+## Note for Docker users
+
+If you are using Docker v18.06 or later, BuildKit mode can be enabled by setting `export DOCKER_BUILDKIT=1` on the client side.
+Docker v18.06 also requires the daemon to be running as the experimental mode.
+
+You need to use `docker build` CLI instead of `buildctl` CLI mentioned in this document.
+See [the `docker build` document](https://docs.docker.com/engine/reference/commandline/build/) for the usage.
+
+Note that not all upstream BuildKit features are available in Docker BuildKit-mode.
```

Comment:
What do you have in mind here? As this doc is describing only Dockerfile it should be the same.

Reply:
deleted